sources/commands/option_pwd.c

```diff
@@ -17,8 +17,8 @@ int	option_pwd(t_control control, t_treenode *node, t_param *param)
 	if (*control.found || !control.choice)
```

Comment:
found is to control of the command has found. remind me please what for is choice?

Reply:
https://github.com/ioannnikulin/minishell/pull/53#discussion_r1786005550